src/main/java/dk/cngroup/lentils/controller/FinalPlaceController.java

```diff
@@ -29,6 +31,9 @@ public FinalPlaceController(final FinalPlaceService finalPlaceService) {
     @GetMapping(value = "/")
     public String finalPlace(final Model model) {
         model.addAttribute("finalPlace", finalPlaceService.getFinalPlace());
+        model.addAttribute(
+                "serverTime",
+                LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss")));
         return VIEW_FINALPLACE_FORM;
```

Comment:
Budou se zobrazovat i sekundy??

Reply:
Asi staci jen minuty, ale to je detail asi ;)

Reply:
Nechal jsem teda jen minuty.